descriptors/ModuleDescriptor-template.json

```diff
@@ -158,7 +168,8 @@
         "invoice.invoice-lines.item.put",
         "invoice.invoice-lines.item.delete",
         "invoice.invoice-number.item.get",
-        "invoice.invoice-number.item.post"
+        "invoice.invoice-number.item.post",
+        "invoice-storage.invoices.item.get"
```

Comment:
Please remove this permission from this section because this one belongs to storage module.

Reply:
Done